src/modules/my-positions/selectors/my-positions.js

```diff
@@ -10,5 +10,9 @@ export default function () {
 export const selectPositionsMarkets = createSelector(
   selectMarkets,
   selectAccountPositionsState,
-  (markets, positions) => (markets || []).filter(market => Object.keys(positions || {}).find(positionMarketID => market.id === positionMarketID))
+  (markets, positions) => {
+    if (!markets || markets.length === 0) return []
+    if (!positions || positions.length === 0) return []
```

Comment:
Is `positions` an array or an object?  The previous code had `Object.keys(positions || {})`, which I'm assuming means that it is an object.